yamlkt/build.gradle.kts

```diff
@@ -13,129 +8,53 @@ plugins {
-            //kotlinx-serialization-core-tvosarm64/                            -         -      
-            //kotlinx-serialization-core-tvossimulatorarm64/                   -         -      
-            //kotlinx-serialization-core-tvosx64/                              -         -      
-            //kotlinx-serialization-core-watchosarm32/                         -         -      
-            //kotlinx-serialization-core-watchosarm64/                         -         -      
-            //kotlinx-serialization-core-watchossimulatora.../                 -         -      
-            //kotlinx-serialization-core-watchosx64/                           -         -      
-            //kotlinx-serialization-core-watchosx86/            
-            // Commented ones are not supported by kotlinx-coroutines-core
-            val nativeTargets: List<String> = arrayOf(
-                // Tier 1:
-                "linuxX64",
-                "macosX64",
-                "macosArm64",
-                "iosSimulatorArm64",
-                "iosX64",
-
-                // Tier 2:
-                "linuxArm64",
-//                "watchosSimulatorArm64",
-                "watchosX64w",
-                "wwatchosArm32",
-                "watchosArm64",
-                "tvosSimulatorArm64",
-                "tvosX64",
-                "tvosArm64",
-                "iosArm64",
-
-                // Tier 3:
-//                "androidNativeArm32",
-//                "androidNativeArm64",
-//                "androidNativeX86",
-//                "androidNativeX64",
-                "mingwX64",
-//                "watchosDeviceArm64",
-
-                // Deprecated:
-                "iosArm32",
-                "watchosX86",
-//                "wasm32",
-                "mingwX86",
-                "linuxArm32Hfp",
-//                "linuxMips32",
-//                "linuxMipsel32",
-            ).flatMap { it.split(", ") }
-            presets.filter { it.name in nativeTargets }
-                .forEach { preset ->
-                    val target = targetFromPreset(preset, preset.name)
-                    nativeMainSets.add(target.compilations[org.jetbrains.kotlin.gradle.plugin.KotlinCompilation.MAIN_COMPILATION_NAME].kotlinSourceSets.first())
-                    nativeTestSets.add(target.compilations[org.jetbrains.kotlin.gradle.plugin.KotlinCompilation.TEST_COMPILATION_NAME].kotlinSourceSets.first())
-                }
-
-            sourceSets {
-                if (!ideaActive) {
-                    configure(nativeMainSets) {
-                        dependsOn(sourceSets.maybeCreate("nativeMain"))
-                    }
-
-                    configure(nativeTestSets) {
-                        dependsOn(sourceSets.maybeCreate("nativeTest"))
-                    }
-                }
+    jvmToolchain(8)
+    jvm {
+        withJava()
+    }
+    js {
+        compilations.all {
```

Comment:
This has no effect for Kotlin only libraries, only when using from JS directly. Is this intentional?

Reply:
I didn't touch that, it was already present before, so I don't know

Reply:
Yes, I know that was general question.